xbmc/epg/EpgDatabase.cpp

```diff
@@ -181,13 +199,14 @@ int CEpgDatabase::Get(CEpgContainer &container)
     {
       while (!m_pDS->eof())
       {
-        int iEpgID                 = m_pDS->fv("idEpg").get_asInt();
-        std::string strName        = m_pDS->fv("sName").get_asString().c_str();
-        std::string strScraperName = m_pDS->fv("sScraperName").get_asString().c_str();
```

Comment:
please revert these changes, this isn't really performance critical if we look-up the columns by name